diff --git a/.gitignore b/.gitignore
index 84014850bb1d2bc407f6b62134e919ab0f5bc0f7..5a2ffd678fbd6a59f8d85a95cebff72af3cb582e 100644
--- a/.gitignore
+++ b/.gitignore
@@ -47,3 +47,21 @@ examples/example_glut_opengl2/example_glut_opengl2
 examples/example_null/example_null
 examples/example_sdl_opengl2/example_sdl_opengl2
 examples/example_sdl_opengl3/example_sdl_opengl3
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/.cmake/api/v1/query/client-MicrosoftVS/query.json
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/binary_dir/src/cmake_install.cmake
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/binary_dir/src/glfw_config.h
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/binary_dir/src/glfw3.pc
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/binary_dir/src/glfw3Config.cmake
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/binary_dir/src/glfw3ConfigVersion.cmake
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/binary_dir/cmake_install.cmake
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/CMakeFiles/3.17.20032601-MSVC_2/CompilerIdC/CMakeCCompilerId.c
+*.cmake
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/CMakeFiles/3.17.20032601-MSVC_2/CMakeDetermineCompilerABI_C.bin
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/CMakeFiles/3.17.20032601-MSVC_2/CMakeDetermineCompilerABI_CXX.bin
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/CMakeFiles/3.17.20032601-MSVC_2/CompilerIdCXX/CMakeCXXCompilerId.cpp
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/CMakeFiles/ShowIncludes/main.c
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/CMakeFiles/ShowIncludes/foo.h
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/CMakeFiles/cmake.check_cache
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/CMakeFiles/TargetDirectories.txt
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/CMakeCache.txt
+examples/example_glfw_vulkan/out/build/x64-Debug (default)/VSInheritEnvironments.txt
